.github/workflows/release.yml

```diff
@@ -112,6 +112,13 @@ jobs:
             rustembedded/cross:${{ matrix.target }} \
             aarch64-linux-gnu-strip \
             /target/${{ matrix.target }}/release/$BIN_NAME
+
+      - name: Build deb package (linux only)
+        if: matrix.build == 'x86_64-linux'
+        run: |
+          cargo install cargo-deb
+          cargo deb --target ${{ matrix.target }}
```

Comment:
The `--deb-version` might be wiser here: elsewhere, versions are derived from the _tag_, not the Cargo version.

Reply:
Using `--deb-version` (which makes the path a little more deterministic too).